### ISSUE TITLE
Add /etc/apt/sources.list Debian configuration Lexer

### ISSUE THREAD

**Issue by mondeja** [pygments/pygments#1524]
Example for Ubuntu:

```
###### Ubuntu Main Repos
deb http://es.archive.ubuntu.com/ubuntu/ bionic main restricted universe multiverse 
deb-src http://es.archive.ubuntu.com/ubuntu/ bionic main restricted universe multiverse 

###### Ubuntu Update Repos
deb http://es.archive.ubuntu.com/ubuntu/ bionic-security main restricted universe multiverse 
deb http://es.archive.ubuntu.com/ubuntu/ bionic-updates main restricted universe multiverse 
deb http://es.archive.ubuntu.com/ubuntu/ bionic-proposed main restricted universe multiverse 
deb http://es.archive.ubuntu.com/ubuntu/ bionic-backports main restricted universe multiverse 
deb-src http://es.archive.ubuntu.com/ubuntu/ bionic-security main restricted universe multiverse 
deb-src http://es.archive.ubuntu.com/ubuntu/ bionic-updates main restricted universe multiverse 
deb-src http://es.archive.ubuntu.com/ubuntu/ bionic-proposed main restricted universe multiverse 
deb-src http://es.archive.ubuntu.com/ubuntu/ bionic-backports main restricted universe multiverse
```

**Comment by chrysle** [comment]
There is already the [`SourcesListLexer`](https://pygments.org/docs/lexers/#pygments.lexers.installers.SourcesListLexer)? Probably a lexer for the newer `ubuntu.sources` file coming with Ubuntu 24.04 should be added instead.

**Comment by sblondon** [comment]
I think this bug report can be closed except if @mondeja explains why SourcesListLexer does not fix his need.

The new [DebianSourcesLexer lexer](https://pygments.org/docs/lexers/#pygments.lexers.installers.DebianSourcesLexer) fills the need to highlight `ubuntu.sources.`

**Comment by mondeja** [comment]
Thanks @sblondon. Closed by #2788
